parquet/src/main/java/org/apache/iceberg/parquet/ParquetAvro.java

```diff
@@ -41,6 +40,74 @@
 
 class ParquetAvro {
 
+  static final GenericData DEFAULT_MODEL =
```

Comment:
it's hard to follow which changes were exactly done in this file due to the large diff